src/plays/smart-converter/styles.css

```diff
@@ -14,8 +14,9 @@
   font-weight: 600;
 }
 .Smart-converter-textarea {
-  width: 100%;
-  margin: auto;
+  width: 100% !important;

```

Comment:
Usage of !imporatnt is not recommended, can you try another solution.

Reply:
There is textarea CSS, which is written in style. css that conflicts with my css So what should I do in this situation?

Reply:
@AbhiPatel10 , I would recommend doing some research on this instead of directly asking for a solution.
Happy learning.

Reply:
Now working fine please have a look on updated code